src/lib/y2storage/planned/assigned_space.rb

```diff
@@ -188,6 +189,20 @@ def require_end_alignment?
         disk_space.require_end_alignment?
       end
 
+      # Whether there are too many partitions to allocate in this space
+      #
+      # @note A disk space holded by an implicit partition table cannot
+      #   allocate more than one partition.
+      #
+      # @see Y2Storage::PartitionTables::ImplicitPt
+      #
+      # @return [Boolean]
+      def too_many_partitions?
```

Comment:
This does not detect all the cases in which there are too many partitions. So the name is confusing. What you are really doing is checking wrong usage of spaces with a true `#partition_to_reuse?` (sorry, I used my suggested name here). So the method name should say that and not pretend to be smarter.